src/styles/_academy.scss

```
@@ -513,6 +513,27 @@
 .grading-overview-filterable-btns {
   padding: 0;
 
+  // Hides overflowed text for the buttons
+  &,
+  p {
+    text-overflow: ellipsis;
+    overflow: hidden;
+
+    // For general filterable buttons
+    & {
+      max-width: 20vw;
+    }
```

Is this not the same as just

```suggestion
    max-width: 20vw;
```
?

---

It is indeed the same, but I separated it for clarity in case it clashes with ```p```

As for being hacky, I agree it's rather hacky, but by any chance do you have a suggestion for it? I'll try to think of an alternative but the implementation using tremor makes it a bit hard to make any custom changes 😭

---

Not at the moment, no, but I'll take a closer look over the recess week. In the meantime, let's keep this PR open and not merge it in case new ideas come up?

As for Tremor, agree. We have had discussions on moving out of Tremor – this might be coming soon, so don't worry too much for now.